contracts/test/strategies/meta.fork-test.js

```diff
@@ -0,0 +1,283 @@
+const { expect } = require("chai");
+
+const { loadFixture } = require("ethereum-waffle");
+const { isForkTest, units, ousdUnits } = require("../helpers");
+const {
+  withBalancedMetaPool,
+  withCRV3TitledMetapool,
+  withOUSDTitledMetapool,
+} = require("../_metastrategies-fixtures");
+
+// Ugly hack to avoid running these tests when running `npx hardhat test` directly.
+// A right way would be to add suffix to files and use patterns to filter
+const forkDescribe = isForkTest ? describe : describe.skip;
+
+forkDescribe("Convex 3pool/OUSD Meta Strategy", function () {
+  this.timeout(0);
+
+  describe("Balanced metapool", () => {
+    let fixture;
+    beforeEach(async () => {
+      fixture = await loadFixture(withBalancedMetaPool);
+    });
+
+    describe("Mint", function () {
```

Comment:
Reading this function for the first time almost brings tear to my eyes. Being able to run these types of checks on every commit is beyond awesome!